packages/mobile/src/identity/verification.ts

```diff
@@ -286,20 +286,28 @@ export async function requestNeededAttestations(
     numAttestationsRequestsNeeded
   )
 
-  await sendTransaction(approveTx, account, TAG, 'Approve Attestations')
```

Comment:
Another possible optimization: cache the attestation fee so we don't have to get it 3 times when calling `makeApproveAttestationFeeTx` above. That function can be made sync.

Reply:
Could you elaborate? As far as I can tell we only call `makeApproveAttestationFeeTx` once?

Reply:
Oh right! confused it with the 3x reveals for a sec. Nvm